development/page-reference/back-office/order/add-new-order/_index.md

```diff
@@ -46,7 +46,7 @@ When you enter the page, first thing you will notice is the `Customer` block:
 {{< figure src="./img/customer-search-block.png" title="Customer search block" >}}
 
 Use the search to find desired customer by `email` or `name` or `create a new one` by pressing `Add new customer`. Customer search is performed using `ajax` by calling [CustomerController::searchAction](https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php). Javascript code can be found in [customer-manager.js](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/order/create/customer-manager.js) 
```

Comment:
We should use frozen code for such links, not branches that have their content evolve

I suggest using
https://github.com/PrestaShop/PrestaShop/blob/1.7.8.0
instead of
https://github.com/PrestaShop/PrestaShop/blob/develop